+ Added ErrorWindow to inform an error to the user: i.e like mods not…

admiralnelson opened the pull request:
ErrorWindow should spawn in case of mod error (during loading or runtime)

![img](https://media.discordapp.net/attachments/677511186295685150/698150270105813012/unknown.png?width=1046&height=588)

admiralnelson committed:
+ Added ErrorWindow to inform an error to the user: i.e like mods not launching, mod error during runtime.

admiralnelson committed:
* Some DLL errors are still not detected properly. It's fixed now.

gallexme commented on Program.cs:
```diff
@@ -45,13 +46,15 @@ static Program()
             AppDomain.CurrentDomain.AssemblyResolve += Program.OnAssemblyResolve;
-            if (!AppDomain.CurrentDomain.FriendlyName.EndsWith("vshost.exe"))
-            {
+            //if (!AppDomain.CurrentDomain.FriendlyName.EndsWith("vshost.exe"))
+            //{
+
                 AppDomain currentDomain = AppDomain.CurrentDomain;
                 currentDomain.UnhandledException +=
                     new UnhandledExceptionEventHandler(OnUnhandledException);
+                currentDomain.FirstChanceException += FirstChanceException;
                 Application.ThreadException += OnUnhandledThreadException;
-            }
+            //}
```
maybe leave that in, it makes sure that visual studio / other debuggers handle the exceptions if its been launched from a debugger,  instead of any error handler from us, less interferences

admiralnelson replied:
For some reason my debugger doesn't run with "vshost.exe". Maybe we should #if !DEBUG instead?

gallexme committed:
do not include user config changes

gallexme reviewed (commented):
think we should wait until Flayan Refactored Project Structure a bit, then ill/you can merge the changes in, good job!

admiralnelson committed:
* temp fixed "Illegal characters in path." error for Bannerlord tweak mod, idk why. That mod runs OK, though.

admiralnelson committed:
Merge branch 'master' of https://github.com/admiralnelson/CommunityLauncher

admiralnelson committed:
* typo in line 148

FlayaN committed:
Update to new csproj project and modio.net project reference (#7)

* - Add modio.net as a submodule
- Move submodule to its own dll
- Use new csproj format

* Fix name of assembly

gallexme committed:
Refactored csproj Settings & Builds see developer-sample.props, added Publish Folder

gallexme committed:
Add Debug Build Config + Missing PostBuild Action

gallexme committed:
Switch Back to Mod Tab After Installation of Mods + Select em

gallexme committed:
De
Delete NTFS ADS Zone.Identifier if it exists(Unblocks DLL/mod if it has a  from Internet Zone)

gallexme committed:
Refactor: Use mod.io SDK's Download Function, Split up some Functions

admiralnelson committed:
* Windows 7 SSL workaround.

admiralnelson committed:
* Rebase project to new project format
* Error window refactoring

admiralnelson committed:
Merge branch 'master' of https://github.com/admiralnelson/CommunityLauncher

# Conflicts:
#	CommunityLauncher.Launcher/Program.cs
#	CommunityLauncher.Launcher/ViewModels/ModVM.cs
#	CommunityLauncher.csproj

admiralnelson commented:
+ Code refactoring for newer project structure

admiralnelson committed:
- Removed old form files
